src/components/POLine/POLineInvoices/POLineInvoicesContainer.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import {
+  filter,
+  find,
+  get,
+} from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Accordion } from '@folio/stripes/components';
+
+import {
+  INVOICE_LINES,
+  INVOICES,
+  RECEIVING_HISTORY,
+} from '../../Utils/resources';
+
+import POLineInvoices from './POLineInvoices';
+
+class POLineInvoicesContainer extends Component {
+  static manifest = Object.freeze({
+    pieces: {
+      ...RECEIVING_HISTORY,
+      GET: {
+        params: {
+          query: (queryParams, pathComponents) => (
+            `checkin == true and poLineId==${pathComponents.lineId || pathComponents.id}`
```

Comment:
this looks confusing `pathComponents.lineId || pathComponents.id`, could you please describe why it's the way it is?

Reply:
It depends on the screen (Orders list -> Oder details -> POL or Lines list -> POL)